indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -152,6 +152,27 @@
   }
 }
 
+:global(.regform-section > .section-sortable-handle) {
```

Comment:
Time to add a styleName to avoid global styles? Eventually all the still-in-use CSS should go into the module anyway.

Reply:
If you go for this keep the old css as-is and just add the new one here (so the legacy version keeps it, hopefully we can then get rid of the whole old css file when removing angular)

Reply:
Added the styleName and restored the old css styles :+1: